yarn.lock

```diff
@@ -3390,6 +3390,14 @@
   dependencies:
     "@types/node" "*"
 
+"@types/cookie-session@^2.0.44":
+  version "2.0.44"
+  resolved "https://packages.aa.com/artifactory/api/npm/npm-public/@types/cookie-session/-/cookie-session-2.0.44.tgz#86f3810bf19716ff716e08f877ce40cee9062267"
```

Comment:
This is why your lint is failing; you need to remove this dependency and then re-add it after modifying your registry... if you need help with that, let me know! We might want to modify the `.npmrc` file to set the registry manually to the standard npm one